docs/long-running-response-status-code.md

```diff
@@ -18,13 +18,28 @@ A '{0}' operation '{1}' with x-ms-long-running-operation extension must have a v
 
 ## Description
 
-For ARM spec, the allowed response status codes for a long DELETE operation are "200" & "204"; the allowed response status codes for a POST operation are "200", "201" ,"202", & "204"; the allowed response status codes for a PUT/PATCH operation are "200" & "201".
-For Data plane spec, the allowed response status codes for a long DELETE operation are "200","202", & "204"; the allowed response status codes for a POST operation are "200", "201" ,"202", & "204"; the allowed response status codes for a PUT/PATCH operation are "200","201", & "202".
+For **ARM (Control Plane) specs**, valid response status codes for long-running (LRO) operations are as follows:
+
+| Operation Name | Valid Response Codes           |
+| -------------- | ------------------------------ |
+| Delete         | `200` and `204`                |
+| Post           | `200`, `201`, `202`, and `204` |
```

Comment:
>`201 [](http://example.com/codeflow?start=26&length=4)

201 is not a valid status code for Post

Reply:
I thought it is:

https://stackoverflow.com/questions/1860645/create-request-with-post-which-response-codes-200-or-201-and-content

Reply:
Consolidating all the review comments here. For context, I only converted the existing documentation into a table and went with that for the rule. However, for control plane do we want it to be this instead?

| Operation Name | Valid Response Codes           |
| -------------- | ------------------------------ |
| LRO Delete         | `200`, `202`, and `204`                |
| LRO Post           | `200`, `202`, and `204` |
| LRO Put            | `200` and `201`                |
| LRO Patch          | `200`               |

Changes:
- Added `202` for LRO delete
- Removed `201` for LRO post
- Removed `201` for LRO patch